release-team/role-handbooks/communications/README.md

```diff
@@ -180,7 +199,7 @@ This is an example of a typical release cycle and the order of how tasks will fl
         <ul>
         <li>Work with Enhancements and Release Note leads to determine which deliverables are most noteworthy post-Enhancements Freeze
         <li>In the coming weeks, follow the progress of these enhancements, as they will roll into the Major Themes and be called out and described in the release blog
-        <li>Work with the Release Team and decide if the release warrants a mid-cycle 'Deprecations and Removals' blog
+        <li>Work with the Release Team and decide if the release warrants a mid-cycle 'Deprecations and Removals' blog. Generally, this is decided around a minimal of 2 deprecations and removals, or if there are significant deprecations and removals that will impact the community. 
```

Comment:
```suggestion
        <li>Work with the Release Team and decide if the release warrants a mid-cycle 'Deprecations and Removals' blog. Generally, this is decided at a minimum of two deprecations or removals, or if there are significant deprecations and removals that will impact the community. 
```

Reply:
Do we want to clarify this further to say that any deprecation or removal that has the potential to cause widespread confusion or panic regardless of its actual impact should justify a mid-cycle deprecations and removals blog? Fully okay with a no on that one, I tend to err way on the side of caution